### ISSUE TITLE
Github-pages out of date

### ISSUE THREAD

**Issue by MidnightJava** [vrapper/vrapper#808]
I have vrapper in my RCP app, and today the sourceforge-hosted p2 update site is not reachable, so my dev environment is broken because I can't load my target or run the maven build. Searching, I found that the update site was hosted on github-pages since sourceforge web access is unreliable. That's great, but the github pages site is pretty far behind, currently at 0.58 vs. 0.72 on soruceforge. ~And I think I don;t have all the features I was installing from the sourceforge site~ EDIT: I was mistaken about missing features. Is the github-pages update site being supported? IF so, can we get it synchronized with the soruceforge site?

**Comment by keforbes** [comment]
@MidnightJava, sorry about that.  The sourceforge update site is still the one I keep current.  I had uploaded the zip file to the github-pages site as a workaround for someone whose corporate network was blocking sourceforge.  As an alternative, you can download the zip file from our Releases page:
https://github.com/vrapper/vrapper/releases/download/0.72.0/vrapper_0.72.0_20170311.zip

**Comment by keforbes** [comment]
Sorry, just to clarify... are you implementing an RCP app that uses vrapper as a dependency?  Mind if I ask what your RCP app does?

**Comment by keforbes** [comment]
Ok, I get it now.  Sourceforge is actually down for maintenance, *that's* why you were asking me to update vrapper.github.io.  I've updated that update site (https://vrapper.github.io/update-site/vrapper-stable/) to host 0.72.0.  Sorry for the confusion, you're good to go now. 👍

**Comment by MidnightJava** [comment]
Thanks for updating the github repo. I wasn't sure if it was sourceforge or the vrapper project that was changing repos. Then I found the issue where people were saying the sourceforge download was not always available.

To answer your question, I'm developing an RCP app under a government contract. It's an IDE that provides development support for a legacy processing framework that's unique to our domain. Some of our developers are resistant to using an IDE, and prefer to just use vim. I added vrapper to the latest release, to enable vim fans to continue to use it while getting the additional benefits provided by the IDE.
